packages/sst/src/node/graphql/index.ts

```diff
@@ -51,12 +55,14 @@ export function GraphQLHandler<C>(config: GraphQLHandlerConfig<C>) {
     };
 
     const { operationName, query, variables } = getGraphQLParameters(request);
+    const validationRules = config.disableIntrospection ? [...specifiedRules, NoSchemaIntrospectionCustomRule] : [specifiedRules];
```

Comment:
Should `specifiedRules` either be spread or just returned falsy expression?

Reply:
Thanks for the catch @jensonb. Updated the PR.